working_with_fhir/working_with_fhir_parsing.bal

```diff
@@ -1,22 +1,17 @@
+import ballerina/io;
 import ballerinax/health.fhir.r4.parser as fhirParser;
 import ballerinax/health.fhir.r4 as fhir;
-import ballerina/io;
-
-public function main() returns error? {
 
-    // The following example is a simple serialized Patient resource to parse
-    string input = "{" +
-    "\"resourceType\" : \"Patient\"," +
-    "  \"name\" : [{" +
-    "    \"family\": \"Simpson\"" +
-    "  }]" +
-    "}";
+final string patientData = string `{"resourceType" : "Patient", "name" : [{"family" : "Simpson"}]}`;
```

Comment:
It was said better to use `json` instead of `string` as this is a json. In #6, I have done that change.